service/cachemdw/cache.go

```diff
@@ -98,9 +98,15 @@ func IsCacheable(
 		return blockNumber > 0 || blockNumber == decode.BlockTagToNumberCodec[decode.BlockTagEarliest]
 	}
 
+	if decode.IsMethodStatic(req.Method) {
+		return true
+	}
+
 	return false
 }
 
+// TODO(yevhenii): add GetTTL method
```

Comment:
right because for truly static methods the response is never going to change so setting an infinite ttl is fine (we could also source the response value from an environment variable and fully bypass the overhead of talking to redis - but I like the simpler approach as long as we can keep the cache from getting full

Reply:
Method is implemented:
```go
func (c *ServiceCache) GetTTL(method string) (time.Duration, bool, error) {
	if decode.MethodHasBlockNumberParam(method) {
		return c.config.CacheMethodHasBlockNumberParamTTL, c.config.CacheMethodHasBlockNumberParamIndefinitely, nil
	}
	...

	return 0, false, ErrRequestIsNotCacheable
}
```